website/src/content/deployment-examples/azure.mdx

```diff
@@ -0,0 +1,103 @@
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Microsoft Azure](https://azure.microsoft.com/free/) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Microsoft Azure project. You can keep the standard Azure URL `https://<project>.1.azurestaticapps.net` or provide your own custom domain.
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<project>.1.azurestaticapps.net"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<project>.1.azurestaticapps.net"
+    }
+  }
+}
+```
+
+# Connect Microsoft Azure with GitHub Actions
+
+The easiest way to deploy to Microsoft Azure is to use [Static Web Apps](https://azure.microsoft.com/en-us/services/app-service/static/#overview) deployment service. This service enables first-class GitHub integration.
+
+Follow the steps in the [Microsoft Azure Static Web App creator](https://portal.azure.com/#create/Microsoft.StaticApp) to create a new project and select `GitHub` as the deployment source.
+
+<Info>
+
+Make sure to grant Microsoft Azure access to your repository in the following step.
+
+</Info>
+
+## Configuring build settings
+
+In the Static Web App setup process you need to configure the following things:
+
+* Select your organization, repository and branch.
+* In the **Build details** section select `Custom` as the build preset.
+* Provide the following build settings:
+    * App location: `/`
+    * Output location: `public`
+
+<Info>
+
+If your Custom Application config [requires environment variables](#using-environment-variables), make sure to provide them in your **Home > Static Web Apps > Your project name > Configuration > Application Settings**.
+
+Alternatively, when using the [GitHub Action workflow file](https://docs.microsoft.com/en-us/azure/static-web-apps/build-configuration?tabs=github-actions#environment-variables) you need to provide the environment variables there.
+
```

Comment:
I'm a little bit confused here since we say this paragraph is an alternative, but it seems it's mandatory (as it uses the wording **you need to provide**).
I understand you need to first configure your env variables following the instruction above and then you must update your Github flow configuration file.
Is it like that or does it work any other way?

Reply:
Thanks, I'm sorry for the confusion. I explained it [here](https://github.com/commercetools/merchant-center-application-kit/pull/2715#discussion_r930804506). That is a good point and I will rewrite that part.